style.css

```diff
@@ -0,0 +1,30 @@
+html{
+    max-width: 100%;
+}
+body{
+    max-width: 100%;
+    background-color: gray;
+    color: blue;
+}
+
+div{
+    max-width: 100%;
+}
+img{
+    max-width: 100%;
+}
+div#cont.cont{
+    display: flexbox;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    max-width: 100%;
+}
+.tile {
+    display: flex;
```

Comment:
I see you used a display `flex`, the ` h2` title of your project section is not positioned at the top, I think you should add the `flex-direction: column` to correct that and make your Project `h2` title aligned at the right position.

![Screenshot 2022-09-26 125206](https://user-images.githubusercontent.com/103499139/192270527-a0452537-b08b-41ff-b0c2-c91aec81ce37.png)